CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- [CustomCommand] new flag `trace` that set the `jaeger-debug-id` header automatically
```

Comment:
I'm thinking that maybe our users that read the changelog wouldn't understand the `CustomCommand` tag, maybe something like:
```suggestion
- [All Commands] new flag `--trace` that sets the `jaeger-debug-id` header automatically on all requests.
```